test-project/Assets/Generated/Source/improbable/gdk/tests/ExhaustiveMapKey.cs

```diff
@@ -511,6 +522,163 @@ public struct Snapshot : ISpatialComponentSnapshot
 
         public static class Serialization
         {
+            public static void SerializeComponent(Improbable.Gdk.Tests.ExhaustiveMapKey.Component component, global::Improbable.Worker.Core.SchemaObject obj, global::Unity.Entities.World world)
+            {
+                {
+                    foreach (var keyValuePair in component.Field1)
```

Comment:
Formatting

Reply:
Sorry I might be blind.. what formatting are you referring to?

Reply:
The `foreach` statement is indented, but its body is one tab stop too shallow

Reply:
Ah right I didn't expand the diff entirely. I think I can driveby that.. although its odd that its doing that 🤔

Reply:
Fixed! 👍